auctions/templates/addon/base.html

```diff
@@ -39,11 +39,11 @@
              </a>
          </div>
 
-        {% if active_issue %}
+        {# {% if active_issue %}
```

Comment:
You have to use `{% comment %}` ... `{% endcomment %}` for multi-line comments.

Reply:
<img width="265" alt="image" src="https://user-images.githubusercontent.com/71928/35311567-c9ef36ea-007c-11e8-871f-41c1034fe2a4.png">

Reply:
We did say that but the lightbulb change wasn't supposed to make it to production.  I'll delete it for now.